### DISCUSSION TITLE
αγγελία για δημιουργία ομάδας

### DISCUSSION THREAD

**Discussion by dimidimots** [sdy60/syllabus#41] [Q&A] [answered]
Πολύ καλησπέρα σας συνάδελφοι,

Μαζί με τον Βασίλη Αγγελίδη έχουμε πει να είμαστε μαζί στην ίδια ομάδα (αφού γνωριζόμαστε από τη σχολή).
Θα ήθελε κανείς να συμμετάσχει παρακαλώ πολύ? ώστε να μην είμαστε μόνο οι 👯‍♂️ μας? 

(Επισήμανση : Λόγω οικογενειακής κατάστασης, ωραρίων, προσωπικών θεμάτων κ.ο.κ θα έχουμε επικοινωνία κυρίως **ασύγχρονη** .)

Ευχαριστώ πολύ! 
Αν θέλετε στείλτε μας μαιλ, ή απαντήστε εδώ.

"ΑΓΓΕΛΙΔΗΣ ΒΑΣΙΛΕΙΟΣ" <std150657@ac.eap.gr>
"ΔΗΜΟΤΣΗΣ ΔΗΜΗΤΡΙΟΣ" <std150660@ac.eap.gr>

**Comment by Katethoma** [comment, marked as answer]
Καλησπέρα. Θα μπορούσα να συμμετέχω εγώ

**Comment by dimidimots** [comment]
Καλησπέρα Κατερίνα,

Καλωσήρθες, θα στείλουμε οδηγίες για συνεννόηση από αύριο.

Οπότε, αρχίζουμε οι 3 μας. Και ανάλογα με το τι θα γίνει με τις άλλες ομάδες, το βλέπουμε αν θέλει/χρειαστεί να έρθει 4ο άτομο στην δικιά μας ομάδα.

ευχαριστούμε πολύ.

**Reply by Katethoma** [reply]
Εντάξει. Ευχαριστώ !
